app/javascript/packs/application_css.scss

```diff
@@ -10,13 +10,25 @@ footer.primary-footer {
 .admin-navbar {
   .navbar-brand img {
     height: 35px;
-    width: 35px;
   }
   .dropdown-toggle {
     cursor: pointer;
   }
+  // I added this because several elements actually would show up above the navbar
+  z-index: 999;
```

Comment:
Firstly, if you are applying a zindex you use one of [Bootstraps variables](https://getbootstrap.com/docs/4.3/layout/overview/#z-index) for this value?

Secondly, looking at chrome's inspector, it seems like this value isn't the value that is applied, are you sure this is having an effect?